app/tailwind.css

```
@@ -15,13 +15,14 @@
   --color-azure-20: rgba(12, 17, 29, 1);
   --color-azure-25: rgba(18, 25, 42, 1);
 
+  /* TODO: --bacground is not defiend, but it's used in other classes (eg: .from-background).
+     Let's define it here */
   --color-background: hsl(var(--background));
```

After reading the daisy UI issue, from what I understood since we are using a dark them this should be replaced by : 

```suggestion
--background: var(--color-base-100);
--color-background: hsl(var(--background));
```


We use background in `tailwind.ts` line 18 and in `SideBarMenu.tsx` line 128